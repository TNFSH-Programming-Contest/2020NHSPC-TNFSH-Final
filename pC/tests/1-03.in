5 7 3
5 2 143949
1 2 632645
4 1 289008
1 3 937622
5 1 96902
4 2 512636
3 4 83483
