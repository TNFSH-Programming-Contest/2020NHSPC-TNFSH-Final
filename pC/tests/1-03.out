744982 1192238 0 83483 1408179 